firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/AppDistroTestApplication.kt

```diff
@@ -0,0 +1,10 @@
+package com.googletest.firebase.appdistribution.testapp
+
+import android.app.Application
+
+class AppDistroTestApplication : Application() {
+    override fun onCreate() {
```

Comment:
Unrelated to this PR, but should we move more initialization code from `MainActiviry.onCreate()` to here?

Also, I'm curious: Would this also work if we added it to `MainActivity.onCreate()`?

Reply:
Ah, makes sense. Let's explore moving the initialization code here in a separate PR once this one is merged.

Also, I think we should restore the button to launch the second activity so we can test those scenarios. Maybe you can do that when adding the radio buttons.